payment-service/src/main/java/com/ecommercepractice/paymentservice/controller/PaymentController.java

```diff
@@ -0,0 +1,47 @@
+package com.ecommercepractice.paymentservice.controller;
+
+import com.ecommercepractice.paymentservice.models.Bill;
+import com.ecommercepractice.paymentservice.models.CardMessage.body.PaymentTypeInfo;
+import com.ecommercepractice.paymentservice.service.PaymentService;
+import io.swagger.annotations.ApiOperation;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@Controller
+@RequestMapping("/api/v1/payment")
+@Slf4j
+public class PaymentController {
+    @Autowired
+    PaymentService paymentService;
+
+    @ApiOperation(value = "WITHDRAW FROM CARD", notes = "Withdraw an explicit amount from a card account against Card server.")
+    @PostMapping("/card/{amount}")
+    public ResponseEntity<Bill> performCardPayment(@PathVariable String amount, @Valid @RequestBody PaymentTypeInfo paymentTypeInfo){
```

Comment:
Please change this path param to be an attribute of the Request Body.

Reply:
Ok